src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -1400,6 +1400,7 @@ private void EnterParameters(ImmutableArray<ParameterSymbol> parameters)
 
         protected virtual void EnterParameter(ParameterSymbol parameter)
         {
+            Visit(parameter.ExplicitDefaultExpression);
```

Comment:
Why would hitting it for abstract methods matter? I'm not sure I understand. Validating that it doesn't crash is good, but I'm sure that somewhere in our test suite we have an abstract method with a default parameter.